graph/patterns/operations.md

```diff
@@ -2,16 +2,16 @@
 
 Microsoft Graph API Design Pattern
 
-*The operations pattern provides the ability to model a change that impacts multiple resources and can't be effectively modeled by using HTTP methods.*
+*The operations pattern provides the ability to model a change that might impact multiple resources and can't be effectively modeled by using HTTP methods.*
 
 ## Problem
 
-Sometimes when modeling a complex business domain, API designers need to model a business operation that effects multiple resources and needs to be performed as a single unit. Modeling the operation via HTTP methods on each individual resource might be either ineffective or not reflect how it's processed by the backend service. In addition, the operation might produce observable side effects.
+Sometimes when modeling a complex business domain, API designers need to model a business operation that effects one or multiple resources and has additional semantic meaning that cannot be expressed by HTTP methods. Modeling the operation via HTTP methods on each individual resource might be either ineffective or expose internal implementation details. In addition, the operation might produce observable side effects.
```

Comment:
> expose internal implementation details

I have a feeling that workloads' use of actions is actually because the resources are actually leaking implementation details, i.e. that the resource boundaries are the same as internal team boundaries, and because of this there is a lack of cohesion within and between resources, requiring workloads to force that cohesion through an action. 

Probably nothing I just wrote is actionable, but it might be something to think about

Reply:
Conway's law in practice